### PR TITLE
US holidays fix

### PR DESCRIPTION
**Pull request by commenthol**
This includes the following changes:

fix(US-CA): Update Hawaiian state holidays
fix(US-CA): Update California state holidays
fix(US-AK): Renaming Columbus Day to Indigenous Peoples Day
feat(US): Veteran's Day Federal Offices closure

Tests are not passing yet. Would like to get a review first...

### CODE REVIEW

**Comment by commenthol** [comment]
will rebase ...

File: data/countries/US.yaml [hunk 1]
```diff
@@ -413,7 +448,7 @@ holidays:
       MA:
         name: Massachusetts
         zones:
-          - America/New_York
+          - America/Chicago
```
Comment (gerryster) [inline data/countries/US.yaml]: America/New_York was correct here. America/Boston would be even better. What timezone database is this using?

Comment (commenthol) [reply data/countries/US.yaml]: it uses moment-timezone. Please see https://github.com/moment/moment-timezone/blob/develop/data/meta/latest.json

File: data/countries/US.yaml [hunk 2]
```diff
@@ -132,10 +139,23 @@ holidays:
...
+              en: President's Day
           monday before April:
             name:
               en: Seward's Day
+          2nd monday in October:
+						name:
+							en: Columbus Day
+            active:
+	            - to: '2015-01-01'
+          # @source https://gov.alaska.gov/Walker/press-room/full-proclamation.html?pr=7296
```
[the remark this answers is not in the document]
Comment (commenthol) [reply data/countries/US.yaml]: Same for me now. Here is another publication on that topic http://time.com/4070797/alaska-indigenous-peoples-day/

File: data/countries/US.yaml (hunk 1 above)
Comment (commenthol) [reply data/countries/US.yaml]: ok commit is reverted.

File: data/countries/US.yaml [hunk 3]
```diff
@@ -84,6 +85,12 @@ holidays:
       11-11:
         name:
           en: Veterans Day
+      11-11 and if sunday then next monday if saturday then previous friday:
```
[the remark this answers is not in the document]
Comment (commenthol) [reply data/countries/US.yaml]: Will add the link

**Comment by gerryster** [comment]
LGTM. Don't forget to update the Alaska reference to the Time Magazine link.

**Comment by gerryster** [comment]
Re-opening as I accidentally closed the PR.

**Comment by commenthol** [comment]
Merging this PR. I am aware that there is an issue with "note" tag with node@11 on certain US states. 
Needs investigation first.

**Pull request merged**